themes/default/boxes.cjs

```diff
@@ -4,15 +4,17 @@ module.exports = function (theme) {
      * Container theme settings
      */
     '.containing-box': {
-      backgroundColor: theme('colors.background.light'),
+      '--containing-box-background-color': 'hsl(var(--tw-colors-background-light-hs),var(--tw-colors-background-light-l))',
```

Comment:
This looks like the right approach here syntactically. 

The one critique I'd have is the verbosity of all the local custom properties I see within all of these CSS rules. Custom properties declared here, within these rules, are not global like they are when declared within `:root {}`. So, you don't have to worry about name collisions. 

That means we can keep them very short, which is something I'd recommend for easier reading and management. As of right now, the line lengths are really long and there's a lot of characters per property reducing readability and adding to potential for mistakes.

For example: rather than this:

```js
'.containing-box': {
  '--containing-box-background-color': 'hsl(var(--tw-colors-background-light-hs),var(--tw-colors-background-light-l))',
  backgroundColor: `var(--containing-box-background-color, ${theme('colors.background.light')})`,
  // ...
}
```

I'd recommend this:

```js
'.containing-box': {
  '--bg-color': 'hsl(var(--tw-colors-background-light-hs), var(--tw-colors-background-light-l))',
  backgroundColor: `var(--bg-color, ${theme('colors.background.light')})`,
  // ...
}
```

Reply:
it makes totally sense, thought of giving a better understanding taking into account the scope of each variable. Do you want me to rename all of the variables?

Reply:
working on the renaming of the variabels

Reply:
Yes, please. The current naming is quite verbose and creates quite a bit of noise when reading.